soc/wch/ch32v/qingke_v4/qingke_v4c/Kconfig.soc.ch32v208

```diff
@@ -1,9 +1,13 @@
 # Copyright (c) 2025 MASSDRIVER EI (massdriver.space)
 # SPDX-License-Identifier: Apache-2.0
 
+config SOC_QINGKE_V4C
+	bool
+	select SOC_SERIES_QINGKE_V4
+
 config SOC_CH32V208
 	bool
-	select SOC_SERIES_QINGKE_V4C
+	select SOC_QINGKE_V4C
```

Comment:
That means that we would have both `SOC_QINGKE_V4C` and `SOC_CH32V208` enabled. I don't see why this would cause a problem but I don't think I have seen this on another place ? But if we don't do that we lose the information about V4C / V4F, so I don't know the correct way.

Reply:
Yea me neither :/

Reply:
Correct me if I'm wrong, but is Qingke V4C just the name of the RISC-V core, whereas the CH32V208 name is for the whole SoC based on that core? If so, then we should keep the information about Qingke V4C in the devicetree instead, as we currently don't care about providing info about individual cores in Kconfig

Reply:
We do actually, one of them has different riscv feature string

Reply:
Right, so the V4F-based SoC can just select those features? And if there's more cores on the SoC with incompatible feature sets, then you can replicate the structure that we currently have for SiFive Freedom SoCs which have 1 "smaller" core and 4 "larger" core with differetes ISA strings

Reply:
So like this latest version?

Reply:
For now this doesnt matter much (v203 is V4B, v303 is V4F, V208 is V4C) but it may when we will have another v30x or CH592 (which is V4C and ch32fun seems to be getting support for it), aesthetically i prefer this version vs just having the socs all in qingke_v4

Reply:
> So like this latest version?

Yeah, thanks a lot! :)